### PR TITLE
feat: add initial scaffolding for the blobstream demo

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Adds to: https://github.com/celestiaorg/orchestrator-relayer/issues/613

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Blobstream rollup dev environment
```
Comment (jcstein) [inline demo/README.md:1]:
```suggestion
# Blobstream rollup dev environment

To run this demo, first clone the repository:

```bash
cd $HOME
git clone https://github.com/celestiaorg/orchestrator-relayer.git
cd orchestrator-relayer/demo
```

Then run the docker-compose setup:

```bash
docker-compose up -d
```

**Review by jcstein — DISMISSED** [review]

File: demo/README.md (hunk 1 above)
Comment (jcstein) [reply demo/README.md:1]: this should also have closing code backticks ```

**Review by evan-forbes — COMMENTED** [review]
are we still needing to merge this?

**Comment by SweeXordious** [comment]
its in a separate branch, we can merge it then josh can continue working on it when he's back

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**